snuba/query/allocation_policies/__init__.py

```diff
@@ -323,13 +323,13 @@ def __init__(
         self._default_config_definitions = [
             AllocationPolicyConfig(
                 name=IS_ACTIVE,
-                description="Whether or not this policy is active.",
+                description="Toggles whether or not this policy is active. If inactive, the system will default to a base policy.",
```

Comment:
>  If inactive, the system will default to a base policy

What is the base policy? Think of yourself as oncall getting paged for this at 2am. does this description help you while you're half asleep?

Reply:
From what I understand, the base policy could be different for each type of allocation policy. For example, the `BytesScannedWindowAllocationPolicy` default is `DEFAULT_PASSTHROUGH_POLICY`. In the future, when we eventually add new policies, I'd imagine their defaults would be different?

On a side note:  one recommendation is what is we defined the default/base policy as an attribute of those inheriting `AllocationPolicy` rather than calling them inline?

Reply:
Discussed offline: Will move is_active logical to base `AllocationPolicy` class